examples/speaker_tasks/diarization/conf/msdd_5scl_15_05_50Povl_infer.yaml

```diff
@@ -0,0 +1,57 @@
+name: "MultiscaleDiarDecoder" 
```

Comment:
can you provide a sample line for inference manifest file

Reply:
that input is exactly same as clustering diarizer and this infer yaml file will be shared by 3 different example files, so let's use the global example json line.